audio/espeak-ng/Portfile

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                espeak-ng
+github.setup        espeak-ng espeak-ng 1.50
+version             1.50
+revision            1
+categories          audio
+platforms           darwin
+maintainers         nomaintainer
+license             GPL-3+
+
+description         eSpeak NG is a multi-lingual software speech synthesizer
+
+long_description    eSpeak NG is a multi-lingual software speech synthesizer
+
+homepage            https://github.com/espeak-ng/espeak-ng
+distname            ${version}
+checksums           rmd160  bd3dd4e17863bb1aa3a964517d888ed9997f0e3e \
+                    sha256  f0505366d5391f1291a9c5cc0816461719f46eaed84cfbf57fb81b0dc381de21 \
+                    size    13665736
+
+depends_lib         port:pcaudiolib \
+                    port:sonic
+depends_build       port:rb26-ronn-ng \
+                    port:rb-kramdown \
```

Comment:
Is it necessary to involve two different old ruby versions in the build process? rb-kramdown uses ruby (1.8.6) and rb26-ronn-ng uses ruby26 (2.6.6). ruby27 (2.7.1) is the latest version and should be used if possible. I realize neither kramdown nor ronn-ng are available in rb27 flavors in MacPorts right now so this may be a to-do item for later.

Reply:
rb-ronn-ng only contains one file (README)
rb19-ronn uses ruby 1.9
rb-kramdown uses ruby 1.8 but throws this error:
/opt/local/lib/ruby/vendor_ruby/1.8/kramdown/document.rb:40:in `data_dir': kramdown data directory not found! This is a bug, please report it! (RuntimeError)

...thus the build.args -i -k

Reply:
I found a way to avoid the -i and -k args